Assets/HoloToolkit/Input/Scripts/Interactable.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Base class Interactable GameObjects can inherit from.
```

Comment:
Very vague description. Please make it more specific.
I intend to use Doxygen to generate API docs for the toolkit.